Marlin/src/HAL/HAL_AVR/endstop_interrupts.h

```diff
@@ -161,6 +161,46 @@ void setup_endstop_interrupts( void ) {
     #endif
   #endif
 
+  #if HAS_X2_MAX
```

Comment:
I made a bone head mistake and didn't add X2 and Y2 endstop interrupts to the other HAL configs, could that be causing the TRAVIS errors?

Reply:
Some care needs to be taken to make sure the other HAL configs will work, but without the hardware, or even the build software, I'm not sure you could have done much besides making a draft.

It looks to me like the complaint is that the test is trying to set Z2_MAX_PIN_2 to 2, which isn't a possibility on the teensy. Maybe test with X_MIN_PIN -1 instead?

I'm not sure you (Scott) really need my help at this point. You know so much more about this than I do. But I was curious about it anyway.

Reply:
Well, the compiler error probably only cared that there was no `Z2_MAX_PIN` set. The Travis scripts could previously only set an option already included in a config file. I've added a script to allow appending more options to the end of a config, and I'm using that to set the `Z2_MAX_PIN`. It should clear up that particular compiler complaint.